pages/blog/status-update-47-20.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "AsyncAPI Initiative Status Update (week 47, 2020)"
```

Comment:
I'd change this title to `Happy Birthday AsyncAPI` too.

Reply:
`Happy Birthday AsyncAPI (week 47, 2020)`? to somehow suggest it is more than just birthday?

Reply:
That's ok 👍